installing/installing_rhv/installing-rhv-restricted-network.adoc

```diff
@@ -0,0 +1,87 @@
+[id="installing-rhv-restricted-network_{context}"]
+= Installing a cluster on {rh-virtualization} in a restricted network
+include::modules/common-attributes.adoc[]
+:context: installing-rhv-restricted-network
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a
+customized {product-title} cluster on {rh-virtualization-first} restricted network by creating an internal mirror of the installation release content.
+
+== Prerequisites
+
+The following items are required to install an {product-title} cluster on a {rh-virtualization} environment.
```

Comment:
s/on a {rh-virtualization}/on an {rh-virtualization}

Reply:
Hi Cody, the a refers to "a RHV environment", if the attribute becomes oVirt, then we have a problem, 
how would you resolve this?

Reply:
Good point; I was spelling out "R-H-V" in my head when reading this, but I'm now realizing we typically refer to it as "rev", so `a` makes more sense in that context. Searching the docs repo, it looks like `a {rh-virtualization}` is used more frequently than `an {rh-virtualization}`, so I say we stick with what you had originally and this can be addressed in another PR, if necessary.